README.markdown

```diff
@@ -286,6 +287,14 @@ See the [documentation](./lib/ngx/resp.md) for this Lua module for more details.
 
 [Back to TOC](#table-of-contents)
 
+## ngx.pipe
+
+This Lua module provides Lua API which could be used to spawn processes and communicate with them nonblockingly.
```

Comment:
Maybe:

This module provides a Lua API to spawn processes and communicate with them in a non-blocking fashion.